Orange/widgets/visualize/owheatmap.py

```diff
@@ -462,6 +463,7 @@ class Information(widget.OWWidget.Information):
         discrete_ignored = Msg("{} discrete column{} ignored")
         row_clust = Msg("{}")
         col_clust = Msg("{}")
+        sparse_densified = Msg("Data was converted to dense format")
```

Comment:
This information will not tell anything to a non-technical user. We should either find a way to explain it (e.g. a warning saying that: Showing this data in heat map may require a lot of memory) or, if we can't, just remove the info.